cocos/rendering/custom/define.ts

```diff
@@ -468,36 +469,37 @@ class DescBuffManager {
         }
     }
     getCurrentBuffer (): Buffer {
+        if (this.buffers.length === 1) {
+            return this.buffers[0];
+        }
```

Comment:
Why this check is needed? I think `this.buffers` is correctly resized in `constructor`.

Reply:
This is a determination of the array length, not the internal length of the buffer.

Reply:
I mean we can remove this guard, the rest of the code still works. I think this is an optimization, because when `this.buffers.length === 1`, we don't need to do the modulo?